src/main/resources/application.yml

```diff
@@ -57,6 +56,7 @@ faf-client:
     application_id: 464069837237518357
     small_image_key: faf_logo_small
     big_image_key: faf_logo_big
+    join-url: https://discord.gg/PfGcWas
```

Comment:
U need to make sure u set validity forever and limit unrestricted. U probably did but just making sure

Reply:
yes I did :) thx